<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerLongitudinalPETCTFindingSettingsDialog</class>
 <widget class="QDialog" name="qSlicerLongitudinalPETCTFindingSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>185</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
     <property name="spacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="LabelName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Finding Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="LineEditName">
       <property name="toolTip">
        <string>Rename the Finding here</string>
       </property>
       <property name="maxLength">
        <number>35</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LabelType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Finding Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the type of the Finding</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="ButtonRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove the selected Finding type</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerLongitudinalPETCTModule.qrc">
         <normaloff>:/Icons/delete.png</normaloff>:/Icons/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="ButtonAddType">
       <property name="toolTip">
        <string>Add the new Finding type to the list</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerLongitudinalPETCTModule.qrc">
         <normaloff>:/Icons/add.png</normaloff>:/Icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LabelAddType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add New Type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="ButtonColor">
       <property name="toolTip">
        <string>Select a color for the new Finding type</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Select Color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="LineEditTypeName">
       <property name="toolTip">
        <string>Name the new Finding type</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>35</number>
       </property>
       <property name="placeholderText">
        <string>New type's name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="Line" name="HorizontalLine">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="ctkExpandButton" name="ExpandButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Show/hide Finding type creation input panel</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkExpandButton</class>
   <extends>QToolButton</extends>
   <header>ctkExpandButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerLongitudinalPETCTModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExpandButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>LabelAddType</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExpandButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>LineEditTypeName</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExpandButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ButtonColor</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExpandButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ButtonAddType</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>qSlicerLongitudinalPETCTFindingSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>qSlicerLongitudinalPETCTFindingSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineEditTypeName</sender>
   <signal>returnPressed()</signal>
   <receiver>ButtonAddType</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExpandButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>HorizontalLine</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
